src/state/actions/jobActions.ts

```diff
@@ -0,0 +1,13 @@
+import { fetchJobsData } from "../../api/jobApi"
+import { Dispatch } from 'redux'
+import { getJobsAction, getJobsLoading, GET_JOBS, GET_JOBS_ERROR, GET_JOBS_LOADING } from "../constants/jobConstant";
+
+export const getData = (name: string) => async (dispatch: Dispatch<getJobsAction | getJobsLoading>) => {
+  dispatch({ type: GET_JOBS_LOADING, loading: true });
```

Comment:
Don't need `loading: true` because we don't use it in reducer

Reply:
@letrongbangg , similar to `dispatch({ type: GET_JOBS, payload: jobs, loading: false });` and `dispatch({ type: GET_JOBS_ERROR, payload: "Get data error", loading: false })`

Reply:
@hienntfaba file changed